samples/subsys/video/capture/README.rst

```diff
@@ -7,9 +7,8 @@
 Description
 ***********
```

Comment:
Has this been tested in 1064_evk after the changes to the sample? Do we need a boards overlay file for 1064_evk?

Reply:
yes, it's tested. The board overlay for 1064 evk is in the same PR.